_usage/command-line.md

```diff
@@ -6,14 +6,18 @@ nav_order: 1
 
 ## Using client tools to work with Babelfish
 
-Many people use SQL Server Management Studio (SSMS) on MS Windows to connect to Babelfish. 
+Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
+connect to Babelfish.
 
-However, there is an ever-growing community which prefers to use the command line to work with the server.
+However, there is an ever-growing community which prefers to use the command
+line to work with the server.
 Those users have the following command-line tool choices on Linux:
 
 - sqlcmd: Using the MS SQL interface
 - tsql: Using the FreeTDS command line tool
```

Comment:
We should not document FreeTDS, this will raise expectations that will not be fulfilled. Anyway, the vast majority of SQL Server customers are ion Windows and those are using Microsoft's own connectivity anyway, so they don't need FreeTDS. FreeTDS may be relevant for some Linux-based apps, but that is not the focus of Babelfish. Remove it.